doc-content/enterprise-only/ai/ai-con.adoc

```diff
@@ -47,9 +46,10 @@ Decision Model and Notation (DMN) is a standard established by the OMG for descr
 PMML is the language used to represent predictive models, mathematical models that use statistical techniques to uncover, or learn, patterns in large volumes of data. Predictive models use the patterns that they learn to predict the existence of patterns in new data. With PMML, you can share predictive models between applications. This data is exported as a PMML file that can be consumed by a DMN model. As a machine learning framework continues to train the model, the updated data can be saved to the existing PMML file. This means that you can use predictive models created by any application that can save the model as a PMML file. Therefore, DMN and PMML integrate well.
 
 .Putting it all together
-COMMENT:  Jess, graphic from slide 3, (not the heading)
+image:ai/123_RHPAM_Integrating_AI_1020_overview.svg[]
 
 This illustration shows how predictive decision automation works.
+
 . Business data enters the system, for example data from a loan application.
 . A decision model that is integrated with a predictive model decides whether or not to approve the loan or whether additional tasks are required.
 . A business action results, for example a rejection letter or offer of loan is sent to the customer.
```

Comment:
"A business action results, for example a rejection letter or offer of loan is sent to the customer."

I suggest:

A business action results, for example a rejection letter or a loan offer is sent to the customer.

Also, in the 1st and 3rd bullets, there are commas followed by "for example". But, no comma after "for example". This gets me every time.

Reply:
Well, I did some checking and I think you are right. As you know, I have an abhorrence of over-using commas. Fixed.